code/modules/mob/living/simple_animal/hostile/ordeal/violet.dm

```diff
@@ -540,7 +540,7 @@
 
 	var/obj/effect/pale_eye/eye = null
 	var/pulsating = FALSE
-	var/pulse_damage = 50 // Dealt each second
+	var/pulse_damage = 15 // Dealt each second
```

Comment:
```suggestion
	var/pulse_damage = 30 // Dealt each second
```
15 for pale is too big a nerf imo, primarily because this hurts one of the strongest parts way too much. Most HE+ armor has SOME pale resist and with this number you can survive 3 hits naked from this, without taking too much out of the potency of this attack.